dev-client/src/components/home/SiteMap.tsx

```diff
@@ -30,30 +35,92 @@ const siteFeatureCollection = (
   })),
 });
 
-const SiteCallout = ({site}: {site: Site}): JSX.Element => {
+type SiteCalloutProps = {
+  site: Site;
+  closeCallout: () => void;
+};
+const SiteCallout = ({site, closeCallout}: SiteCalloutProps) => {
+  const {t} = useTranslation();
+  const project = useSelector(state =>
+    site.projectId === undefined
+      ? undefined
+      : state.project.sites[site.projectId],
+  );
+
   return (
     <Mapbox.MarkerView
       coordinate={[site.longitude, site.latitude]}
       anchor={{x: 0.5, y: 0}}
       allowOverlap={true}>
-      <Box backgroundColor="background.default">
-        <Heading size="md">{site.name}</Heading>
-        <Text>latitude: {site.latitude}</Text>
-        <Text>longitude: {site.longitude}</Text>
+      <Box bg="grey.200" padding="4">
+        <Flex direction="row" align="top" justify="space-between">
+          <Heading size="lg">{site.name}</Heading>
+          <MaterialIconButton
+            name="close"
+            onPress={closeCallout}
+            iconProps={{size: 'md'}}
+          />
+        </Flex>
+        {project && <Heading size="md">{project.name}</Heading>}
+        <Box height="4" />
+        <Box>
+          <Text>
+            {t('site.last_updated', {
+              date: 'dd-mm-yyyy',
```

Comment:
question: would yyyy-mm-dd make sense here?

Reply:
i am just following the design here, but all these values are placeholders until the actual functionality gets filled out